odoo/addons/base/wizard/base_translation_wizard.py

```diff
@@ -0,0 +1,90 @@
+from odoo import api, models, fields
+from odoo.exceptions import AccessError
+
+
+class TranslationWizard(models.TransientModel):
+    _name = 'translation.field.wizard'
+    _description = "Translation Field Wizard"
+
+    translation_lines = fields.One2many('translation.lines', 'translation_wizard_id', string="Translations")
+
+    @api.model
+    def default_get(self, fields_list):
+        res = super(TranslationWizard, self).default_get(fields_list)
+        IrTranslation = self.env['ir.translation']
+
+        domain = self._prepare_domain()
+        translations = IrTranslation.search(domain)
+        res['translation_lines'] = [[0, False, {
+            'value': line.value or line.source,
```

Comment:
Is it a good idea to prefill with the source? Is there a reason not to use an empty string?

Reply:
@mart-e We decided to load source as translation value when records are loaded in wizard o2m, if there is nothing in translation then source should be displayed, user can change translation value, do you want empty string?

Reply:
@msh-odoo with an empty string, it is very easy to identify which one still needs translations